Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/EditorModeCommon.azsli

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include <viewsrg.srgi>
+
+#include <Atom/Features/SrgSemantics.azsli> 
+#include <Atom/Features/PostProcessing/FullscreenPixelInfo.azsli>
+#include <Atom/Features/PostProcessing/FullscreenVertex.azsli>
+
+partial ShaderResourceGroup PassSrg : SRG_PerPass_WithFallback
+{
+    //! The color buffer to apply the editor mode feedback effects to.
+    Texture2D<float4> m_framebuffer;
+
+    //! The non-linear depth buffer data for fragments in the color buffer.
+    Texture2D<float4> m_depth;
+
+    //! The entity mask containing the visible fragments of entities of interest.
+    Texture2D<float4> m_entityMask;
+
+    //! The final blend amount that is used to scale the calculated blend values.
+    float m_finalBlendAmount;
+    
+    Sampler LinearSampler
+    {
+        MinFilter = Linear;
+        MagFilter = Linear;
+        MipFilter = Linear;
+        AddressU = Clamp;
+        AddressV = Clamp;
+        AddressW = Clamp;
+    };
+
+    Sampler PointSampler
+    {
+        MinFilter = Point;
+        MagFilter = Point;
+        MipFilter = Point;
+        AddressU = Clamp;
+        AddressV = Clamp;
+        AddressW = Clamp;
+    };
+
+    //! Returns the dimensions of the specified 2D image.
+    float2 GetImageSize(Texture2D<float4> image)
+    {
+        float3 size = float3(0., 0., 0.);
+        image.GetDimensions(size.x, size.y);
+        return size;
+    }
+
+    //! Returns the linear depth value of the specified non-linear depth value.
+    float CalculateLinearDepth(const float zDepth)
+    {
+        return abs(((ViewSrg::GetFarZTimesNearZ()) / (ViewSrg::GetFarZMinusNearZ() * zDepth - ViewSrg::GetFarZ()))); 
+    }
```

Comment:
Same, this doesn't use any members of PassSrg and can be moved outside as a standalone function

Reply:
Will address 👍